source/apis/content-store.html.md.erb

```diff
@@ -0,0 +1,142 @@
+---
+layout: api_layout
+title: Content Store
+parent: /apis.html
+---
+
+The [Content Store][content-store-repo] is the central store for all published content on
+GOV.UK. When given a path, it responds with the content that should be displayed
+on that path.
+
+While it is primarily intended for the use of the frontend apps, it is also
+exposed externally at `https://www.gov.uk/api/content/<path>`, where `<path>` is
+the full path of the item on GOV.UK.
+
+The full list of fields in a document depends on the document type and schema
+name, which are themselves fields included at the top level; these schemas are
+defined in the [govuk-content-schemas repository][schemas-repo]. However, the
+bulk of the fields are common between all schemas, and these are primarily
+contained in three sections, as detailed below.
+
+## Top level fields
+
+These fields are constant across all schemas.
```

Comment:
I think would be good to clarify here that you are not talking about the schemas, which themselves are JSON objects with fields, but about the different types of content. For example, I think `These fields are constant across all document types` is clearer.